src/main/java/frc/robot/commands/ClimberGoTo.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.Constants;
+
+public class ClimberGoTo extends CommandBase {
+  /** Creates a new ClimberFullRetract. */
+  public double endpoint;
+  public double startpoint;
```

Comment:
These members should be private, not public.  They should not be accessible outside of this class.